GOVERNANCE.md

```diff
@@ -0,0 +1,182 @@
+# Overview
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+# Roles and responsibilities
+
+## Users
+
+Users are community members who have a need for the project. They are the most
+important members of the community and without them the project would have no
+purpose. Anyone can be a user; there are no special requirements.
+
+The project asks its users to participate in the project and community as much
```

Comment:
IMHO it is not the project that asks but the community who is asking. Also I would think that using "invite" instead of "ask" would be nicer. Maybe "The community invites all users to participate in the project as a much as possible".